### DISCUSSION TITLE
selective scan, only new albums / folder

### DISCUSSION THREAD

**Discussion by nadigo** [photoview/photoview#702] [General]
Hi, 

I have thousands of files and I'm adding a few hundred every month or so (which I assume is regular behavior for casual photographers) - it would be nice to have a way to only scan the added files (every week / day / month) and to be able to re-scan a folder on the disk in case there was a change on that specific one (without scanning the full album / top-level folder) 

What would be the best way to add a selective scan setting, either to scan only new files or to manually scan albums / folder.

N

**Comment by viktorstrate** [comment]
The best solution would be something like what is proposed in #201
